src/Core/Domain/Product/Command/UpdateProductCommand.php

```diff
@@ -276,6 +276,11 @@ class UpdateProductCommand
      */
     private $active;
 
+    /**
+     * @var DateTimeInterface
```

Comment:
```suggestion
     * @var DateTimeInterface|null
```
All fields in the update command are nullable so the product can be updated partially